CHANGELOG.md

```diff
@@ -5,12 +5,19 @@ x.y.z Release notes (yyyy-MM-dd)
   This allows you to select elements in a collection with a given IndexSet (Cocoa [#7298](https://github.com/realm/realm-cocoa/issues/7298).
 * Add `App.emailPasswordAuth.retryCustomConfirmation(email:completion:)` and `[App.emailPasswordAuth retryCustomConfirmation:completion:]`. 
   These functions support retrying a [custom confirmation](https://docs.mongodb.com/realm/authentication/email-password/#run-a-confirmation-function) function.
+* Improve performance of creating collection notifiers for Realms with a complex schema. 
+  This means that the first run of a synchronous query, first call to observe() on a collection, 
+  or any call to find_async() will do significantly less work on the calling thread.
```

Comment:
The wording on this needs to be adjusted a little. find_async() isn't something we have in Cocoa and we don't distinguish between sync and async queries like Java does, so it should be something like "the first run of a query or first call to observe() on a collection will do significant less work".